WordPress/Classes/ViewRelated/Me/AppSettingsViewController.swift

```diff
@@ -0,0 +1,105 @@
+import Foundation
+import UIKit
+import RxSwift
+import WordPressComAnalytics
+
+func AppSettingsViewController(account account: WPAccount?) -> ImmuTableViewController {
+    return AppSettingsViewController()
+}
+
+func AppSettingsViewController() -> ImmuTableViewController {
+    let controller = AppSettingsController()
+    let viewController = ImmuTableViewController(controller: controller)
+    return viewController
+}
+
+private struct AppSettingsController: SettingsController {
```

Comment:
Since this doesn't interact with remote data (so it's always going to be instantly available), it can be simplified by skipping `ImmuTableViewController`/`RxSwift`. Just use a regular `UITableViewController` with a `ImmuTableViewHandler` like we do in `MeViewController`.